packages/redux/src/entities/schemas/productsList.ts

```diff
@@ -97,8 +105,9 @@ export default new schema.Entity(
         },
       );
       const productsList = {
-        facetGroups,
+        facetGroups: newFacetGroups,
         filterSegments: newFilterSegments,
+        hash,
```

Comment:
This can be removed as it is not necessary for normalisation or consumption from the clients.

Reply:
If we don't return the hash here, then we get an issue in the boilerplate at the `getListingActiveFiltersByFacetGroup ` 🤔

Reply:
You are right, I thought this was not necessary for clients to access it.